crates/install-wheel-rs/src/uninstall.rs

```diff
@@ -233,6 +233,14 @@ pub fn uninstall_legacy_editable(egg_link: &Path) -> Result<Uninstall, Error> {
         })
         .ok_or_else(|| Error::InvalidEggLink(egg_link.to_path_buf()))?;
 
+    let target_line = if cfg!(windows) {
+        // Do the equivalent of os.path.normcase
+        // This comes from `pkg_resources.normalize_path`
+        target_line.replace("\\", "/").to_lowercase()
```

Comment:
Just going off of the non winapi version of https://github.com/python/cpython/blob/a7711a2a4e5cf16b34fc284085da724a8c2c06dd/Lib/ntpath.py#L78